pkgs/development/libraries/librealsense/default.nix

```diff
@@ -13,17 +28,33 @@ stdenv.mkDerivation rec {
     sha256 = "04macplj3k2sdpf1wdjm6gsghak5dzfhi2pmr47qldh2sy2zz0a3";
   };
 
+  enableParalellBuilding = true;
+
   buildInputs = [
     libusb1
-  ];
+  ] ++ pkgs.lib.optionals buildExamples (builtins.attrValues exampleDeps);
 
   nativeBuildInputs = [
     cmake
     ninja
     pkgconfig
   ];
 
-  cmakeFlags = [ "-DBUILD_EXAMPLES=false" ];
+  cmakeFlags = [ ''-DBUILD_EXAMPLES=${if buildExamples then "true" else "false"}'' ];
+
+  # copy udev rules to $out
+  # fix executable paths in udev rules and copy them to $udev_bins_path
+  postInstall = ''
+  cd ..
+  mkdir -p $out/lib/udev/rules.d
+  udev_bins_path=$out/udev-bins
+  mkdir -p $udev_bins_path
+  cp -ra config/{usb-R200-in,usb-R200-in_udev} $udev_bins_path
+  sed -i 's@/bin/bash@/bin/sh@' $udev_bins_path/*
+  chmod +x $udev_bins_path/*
+  cp config/99-realsense-libusb.rules $out/lib/udev/rules.d/99-realsense-libusb.rules
+  sed -i -e "s@/usr/local/bin/\\(usb-R200-in_udev\\|usb-R200-in\\)@$udev_bins_path/\\1@" -e "s@/bin/sh@$(type -p sh)@" $udev_bins_path/* $out/lib/udev/rules.d/99-realsense-libusb.rules
+  '';
```

Comment:
This is good. Please open an issue upstream, fix indentation, and link in a comment here to the issue upstream.